### PR TITLE
Release v2.4.4

### PR DESCRIPTION
**Pull request by dongjoon-hyun**
This aims to update website to release 2.4.4.

### CODE REVIEW

**Comment by dongjoon-hyun** [comment]
Hi, @srowen , @gatorsmile , @cloud-fan , @dbtsai , @felixcheung , @HyukjinKwon , @rdblue , @viirya , @wangyum .
Could you review this?

File: js/downloads.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ var packagesV8 = [hadoop2p7, hadoop2p6, hadoopFree, sources];
 // 2.4.0+
 var packagesV9 = [hadoop2p7, hadoop2p6, hadoopFree, scala2p12_hadoopFree, sources];
 
-addRelease("2.4.3", new Date("05/07/2019"), packagesV9, true);
+addRelease("2.4.4", new Date("08/30/2019"), packagesV9, true);
```
Comment (dongjoon-hyun) [inline js/downloads.js:25]: This is yesterday which the vote passed.

Comment (dongjoon-hyun) [reply js/downloads.js:25]: To reviewers, in the news, I'm using `2019-08-30` although today is 8/31. 
We can change it when this PR is decided to merge. **For now, SparkR is not published yet**.
All the others (Apache dist/ Maven/ PyPI publish) were completed. Some Apache mirrors may be still behind at this moment.

**Review by srowen — APPROVED** [review]

**Comment by dongjoon-hyun** [comment]
Thank you for review and approval, @srowen !

**Review by HyukjinKwon — APPROVED** [review]

File: site/news/spark-2-4-4-released.html [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+          <li class="divider"></li>
+          <li><a href="/third-party-projects.html">Third-Party Projects</a></li>
+        </ul>
+      </li>
+      <li class="dropdown">
+        <a href="#" class="dropdown-toggle" data-toggle="dropdown">
+          Documentation <b class="caret"></b>
+        </a>
+        <ul class="dropdown-menu">
+          <li><a href="/docs/latest/">Latest Release (Spark 2.4.3)</a></li>
```
Comment (viirya) [inline site/news/spark-2-4-4-released.html]: 2.4.3 -> 2.4.4?

File: site/releases/spark-release-2-4-4.html [hunk 3]
```diff
@@ -0,0 +1,250 @@
...
+          <li class="divider"></li>
+          <li><a href="/third-party-projects.html">Third-Party Projects</a></li>
+        </ul>
+      </li>
+      <li class="dropdown">
+        <a href="#" class="dropdown-toggle" data-toggle="dropdown">
+          Documentation <b class="caret"></b>
+        </a>
+        <ul class="dropdown-menu">
+          <li><a href="/docs/latest/">Latest Release (Spark 2.4.3)</a></li>
```
Comment (viirya) [inline site/releases/spark-release-2-4-4.html]: 2.4.3 -> 2.4.4?

Comment (viirya) [inline site/releases/spark-release-2-4-4.html]: There are similar section in other files, like site/releases/spark-release-2-4-1.html:

```
<ul class="dropdown-menu">
  <li><a href="/docs/latest/">Latest Release (Spark 2.4.3)</a></li>
  <li><a href="/documentation.html">Older Versions and Other Resources</a></li>
  <li><a href="/faq.html">Frequently Asked Questions</a></li>
  </ul>
```

Shall we update 2.4.3 there to 2.4.4 too?

File: site/news/spark-2-4-4-released.html (hunk 2 above)
Comment (viirya) [inline site/news/spark-2-4-4-released.html]: In some files like site/news/spark-2-4-1-released.html, there is a similar section:

```
<ul class="dropdown-menu">
  <li><a href="/docs/latest/">Latest Release (Spark 2.4.3)</a></li>
  <li><a href="/documentation.html">Older Versions and Other Resources</a></li>
  <li><a href="/faq.html">Frequently Asked Questions</a></li>
</ul>
```

Shall we update it too?

Comment (dongjoon-hyun) [reply site/news/spark-2-4-4-released.html]: Oh. Thanks.

File: _layouts/global.html [hunk 4]
```diff
@@ -121,7 +121,7 @@
           Documentation <b class="caret"></b>
         </a>
         <ul class="dropdown-menu">
-          <li><a href="{{site.baseurl}}/docs/latest/">Latest Release (Spark 2.4.3)</a></li>
+          <li><a href="{{site.baseurl}}/docs/latest/">Latest Release (Spark 2.4.4)</a></li>
```
Comment (dongjoon-hyun) [inline _layouts/global.html:124]: This was the missing part and the others are regenerated. Thanks, @viirya .

**Review by viirya — APPROVED** [review]

File: releases/_posts/2019-08-30-spark-release-2-4-4.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+  - [[SPARK-27992]](https://issues.apache.org/jira/browse/SPARK-27992): Fix PySpark socket server to sync with JVM connection thread future
+  - [[SPARK-28015]](https://issues.apache.org/jira/browse/SPARK-28015): Check stringToDate() consumes entire input for the yyyy and yyyy-[m]m formats
+  - [[SPARK-28308]](https://issues.apache.org/jira/browse/SPARK-28308): CalendarInterval sub-second part should be padded before parsing
+  - [[SPARK-28489]](https://issues.apache.org/jira/browse/SPARK-28489): KafkaOffsetRangeCalculator.getRanges may drop offsets
+  - [[SPARK-28699]](https://issues.apache.org/jira/browse/SPARK-28699): Cache an indeterminate RDD could lead to incorrect result while stage rerun
+  - [[SPARK-28780]](https://issues.apache.org/jira/browse/SPARK-28780): Spark 2.2 accidentally introduced the method LinearSVCModel.setWeightCol. This method works incorrectly, too. It is deprecated in 2.4.4 and will be removed in 3.0.0.
+
+You can consult JIRA for the [detailed changes](https://s.apache.org/spark-2.4.4).
```
Comment (gatorsmile) [inline releases/_posts/2019-08-30-spark-release-2-4-4.md]: IMO, the right link should be: https://issues.apache.org/jira/secure/ReleaseNote.jspa?projectId=12315420&version=12345466

**Comment by gatorsmile** [comment]
LGTM except one comment.

File: releases/_posts/2019-08-30-spark-release-2-4-4.md (hunk 5 above)
Comment (dongjoon-hyun) [reply releases/_posts/2019-08-30-spark-release-2-4-4.md]: Got it. Thanks.

Comment (dongjoon-hyun) [reply releases/_posts/2019-08-30-spark-release-2-4-4.md]: I updated the link to there.
```
$ curl https://s.apache.org/spark-2.4.4
Go here: https://issues.apache.org/jira/secure/ReleaseNote.jspa?projectId=12315420&version=12345466
```

**Comment by dongjoon-hyun** [comment]
Hi, All. It seems that SparkR needs a little bit more time. According to @felixcheung 's advice, I'll announce this first without waiting for SparkR.

**Pull request merged**